### ISSUE TITLE
VMS: Landmanager Access to Summary Reports in MBP Crew Corner

### ISSUE THREAD

**Issue by WaBirder57** [washingtontrails/vms#370]
Land managers need to be able to see all Summary Reports for work done on trails in their lands.

Can we add a Summary Reports link, as we have for Crew Leaders, on the My Backpack, Crew Corner area so that Land Managers can see all these reports?

Julie reports:
On Story #95 - This method for Land Managers does not work: "Navigate to a work party which has trails for which the user is a land manager." It requires there to be a future, published work party on one of their trail segments in order for them to get in. The land managers often want to look back at these reports at the end of the year, when there will not be any future parties posted on their trail segments. They can't use custom dates to reference a past work party, because past work parties are no longer accessible via Custom Dates after they are switched to "complete." 
I suggest land managers have the "Summary Reports" link in MyBackpack just like the CL's and staff have, and it just show the work parties they have access to. If that's too expensive / complicated, land managers should just have access to ALL summary reports like they do now. Is anyone opposed to that? It's more info than they need to see, and it's oversharing, but the alternative, as it's built now, does not work.

**Comment by davisagli** [comment]
This was almost working, but needed a couple fixes. I think it is working correctly now.

**Comment by ebskopin** [comment]
Per Julie's Re-test:

This does not appear to be fixed. 

https://github.com/washingtontrails/vms/issues/370

"Summary Reports" is not listed in the right column in MyBackpack for Land Managers, which  is one way I recommended the land managers navigate to their summary reports. Its not clear to me is what was decided to be the best way to navigate to summary reports for land managers. (What did David fix?) 

As a land manager, this is how I got to summary reports: 
In MyBackpack, I clicked on "My Work Parties." 
I selected "Completed" from the pulldown menu.
I went down the list clicking on each work party until one had the "Reports" tab. Presumably, this is a work party that is on a trail segment in my jurisdiction. 
I clicked "See all summaries" at the bottom of the report to get to the summaries. From there, I can search summary reports and access reports from other trail segments I'm aware of that are in my jurisdiction. 
This only worked because "Al Pine" was personally registered for work parties. 
**Most land managers will not be personally registered for any work parties, and will not be able to navigate to summaries as I just described.**
